apps/common/controllers/address-lookup-loop.js

```diff
@@ -34,10 +36,23 @@ module.exports = class AddressLookupLoopController extends BaseController {
       items,
       hasAddresses,
       postcode,
-      id
+      id,
+      hasCategories
     });
   }
 
+  translateCategories(req, values) {
+    if (!Array.isArray(values)) {
+      values = [values];
+    }
+    const categories = values.map(value => {
+      let key = `fields.location-address-category.options.${value}.label`;
+      let result = req.translate(key);
+      return result === key ? value : result;
+    }).join('\n');
+    return categories;
+  }
```

Comment:
I'm in danger of repeating myself but, if `categories` is only used once, you could simply return the result of `values.map`

There is quite a lot of duplication of code between this and apps/common/controllers/add-another-address-loop.js